tools/parse_baselines/providers/snapshot_restore.py

```diff
@@ -12,7 +12,7 @@
 # that were not caught while gathering baselines. This provides
 # slightly better reliability, while not affecting regression
 # detection.
-DELTA_EXTRA_MARGIN = 4
+DELTA_EXTRA_MARGIN = 6
```

Comment:
This commit is identical with this one https://github.com/firecracker-microvm/firecracker/pull/3140/commits/a0e4a8a240f49ca42823d3f650276010eced6f96 but I need it here for when I gather and parse baselines.